<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="C:\Documents and Settings\Administrator\My Documents\Visual Studio 2010\Projects\Zaypay\Zaypay\bin\Release\Zaypay.dll" total="4" errors="0" failures="1" not-run="0" inconclusive="0" ignored="0" skipped="0" invalid="0" date="2012-09-24" time="15:02:29">
  <environment nunit-version="2.6.1.12217" clr-version="2.0.50727.3053" os-version="Microsoft Windows NT 5.2.3790 Service Pack 2" platform="Win32NT" cwd="C:\Documents and Settings\Administrator\My Documents\Visual Studio 2010\Projects\Zaypay\Zaypay\bin\Release" machine-name="SHAHZAD-8DD76D1" user="Administrator" user-domain="SHAHZAD-8DD76D1" />
  <culture-info current-culture="en-US" current-uiculture="en-US" />
  <test-suite type="Assembly" name="C:\Documents and Settings\Administrator\My Documents\Visual Studio 2010\Projects\Zaypay\Zaypay\bin\Release\Zaypay.dll" executed="True" result="Failure" success="False" time="0.404" asserts="0">
    <results>
      <test-suite type="Namespace" name="Zaypay" executed="True" result="Failure" success="False" time="0.371" asserts="0">
        <results>
          <test-suite type="Namespace" name="Test" executed="True" result="Failure" success="False" time="0.371" asserts="0">
            <results>
              <test-suite type="TestFixture" name="PriceSettingTest" executed="True" result="Failure" success="False" time="0.371" asserts="0">
                <results>
                  <test-case name="Zaypay.Test.PriceSettingTest.ListLocales" executed="True" result="Success" success="True" time="0.337" asserts="2" />
                  <test-case name="Zaypay.Test.PriceSettingTest.ListLocaleWithException" executed="True" result="Success" success="True" time="0.017" asserts="0" />
                  <test-case name="Zaypay.Test.PriceSettingTest.LocaleForIPWithException" executed="True" result="Failure" success="False" time="0.000" asserts="0">
                    <failure>
                      <message><![CDATA[System.Exception was expected]]></message>
                      <stack-trace />
                    </failure>
                  </test-case>
                  <test-case name="Zaypay.Test.PriceSettingTest.LocalForIP" executed="True" result="Success" success="True" time="0.000" asserts="1" />
                </results>
              </test-suite>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>